cmd/gateway/main.go

```diff
@@ -673,6 +678,9 @@ func startGateway() {
 		if newConfig.dapiProxyServices != config.dapiProxyServices {
 			logger.Warn("config changes for dapiProxyServices require a restart")
 		}
+		if newConfig.dapiDisableAdmin != config.dapiDisableAdmin {
+			logger.Warn("config changes for dapiDisableAdmin require a restart")
+		}
```

Comment:
DRY opportunity for someone.  :)

Reply:
This values are purposely defined as a concrete structure (they are already parsed, had defaults applied, etc... at this point), and thus there isn't much way to avoid the repetition for each field here.